bedrock/settings/__init__.py

```diff
@@ -265,7 +265,13 @@
 
 # Mainly for overriding CSP settings for CMS admin.
 # Works in conjunction with the `bedrock.base.middleware.CSPMiddlewareByPathPrefix` middleware.
-CSP_PATH_OVERRIDES = {}
+CMS_CSP = CONTENT_SECURITY_POLICY.copy()
```

Comment:
passing thought: if we're gonna make this specific to `/cms-admin/images/` do we call it `CMS_IMAGES_CSP` now, or just wait to see if we need any extra CSP and _then_ deal with it? Probably the latter, tbh

Reply:
Is this image loaded from the `/cms-admin/images/` path only? We could also just have a `/cms-admin/` CSP but I feel like that opens up some risk by allowing the wagtail admin to load any blob.

Reply:
Yeah the image only comes up as a blob in `cms-admin/images/` so locking tightly to that is good. I wasn't suggesting we move it to `cms-admin/` but maybe rename the `CMS_CSP` var to `CMS_IMAGES_CSP` but it's total bikeshed territory I'm into now :D

Reply:
Yep, I updated the variable naming. So I'll merge this in.